<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>add_book</class>
 <widget class="QMainWindow" name="add_book">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1421</width>
    <height>965</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add Book</string>
  </property>
  <property name="windowIcon">
   <iconset resource="bookholics.qrc">
    <normaloff>:/image/1.png</normaloff>:/image/1.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>10</y>
      <width>431</width>
      <height>251</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/logo.jpg);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>290</y>
      <width>601</width>
      <height>381</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>582</width>
       <height>572</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="0">
       <widget class="QPushButton" name="button_add_book">
        <property name="font">
         <font>
          <family>Times New Roman</family>
          <pointsize>14</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">padding:4px;</string>
        </property>
        <property name="text">
         <string>Add Book</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="font">
           <font>
            <family>Times New Roman</family>
            <pointsize>11</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Book ID</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="book_id">
          <property name="font">
           <font>
            <family>Times New Roman</family>
            <pointsize>11</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">padding:2px;
border-style:solid;
border:2px solid blue;
border-radius:8px;
</string>
          </property>
          <property name="placeholderText">
           <string>Enter Unique ID</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_2">
          <property name="font">
           <font>
            <family>Times New Roman</family>
            <pointsize>11</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Book Name</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="book_name">
          <property name="font">
           <font>
            <family>Times New Roman</family>
            <pointsize>11</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">padding:2px;
border-style:solid;
border:2px solid blue;
border-radius:8px;</string>
          </property>
          <property name="placeholderText">
           <string>Enter valid Name</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_3">
          <property name="font">
           <font>
            <family>Times New Roman</family>
            <pointsize>11</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Book Keyword</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_4">
          <property name="font">
           <font>
            <family>Times New Roman</family>
            <pointsize>11</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Publication</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLineEdit" name="book_publication">
          <property name="font">
           <font>
            <family>Times New Roman</family>
            <pointsize>11</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">padding:2px;
border-style:solid;
border:2px solid blue;
border-radius:8px;</string>
          </property>
          <property name="placeholderText">
           <string>Enter  Publication name</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_10">
          <property name="font">
           <font>
            <family>Times New Roman</family>
            <pointsize>11</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Author List</string>
          </property>
         </widget>
        </item>
        <item row="8" column="0">
         <widget class="QLabel" name="label_13">
          <property name="font">
           <font>
            <family>Times New Roman</family>
            <pointsize>11</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Price</string>
          </property>
         </widget>
        </item>
        <item row="8" column="1">
         <widget class="QLineEdit" name="book_price">
          <property name="font">
           <font>
            <family>Times New Roman</family>
            <pointsize>11</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">padding:2px;
border-style:solid;
border:2px solid blue;
border-radius:8px;</string>
          </property>
          <property name="placeholderText">
           <string>Enter valid price</string>
          </property>
         </widget>
        </item>
        <item row="9" column="0">
         <widget class="QLabel" name="label_14">
          <property name="font">
           <font>
            <family>Times New Roman</family>
            <pointsize>11</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Quantity</string>
          </property>
         </widget>
        </item>
        <item row="9" column="1">
         <widget class="QLineEdit" name="book_quantity">
          <property name="font">
           <font>
            <family>Times New Roman</family>
            <pointsize>11</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">padding:2px;
border-style:solid;
border:2px solid blue;
border-radius:8px;</string>
          </property>
          <property name="placeholderText">
           <string>Enter valid Quantity</string>
          </property>
         </widget>
        </item>
        <item row="10" column="0">
         <widget class="QLabel" name="label_17">
          <property name="font">
           <font>
            <family>Times New Roman</family>
            <pointsize>11</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>ISBN NO.</string>
          </property>
         </widget>
        </item>
        <item row="10" column="1">
         <widget class="QLineEdit" name="book_isbn_no">
          <property name="font">
           <font>
            <family>Times New Roman</family>
            <pointsize>11</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">padding:2px;
border-style:solid;
border:2px solid blue;
border-radius:8px;</string>
          </property>
          <property name="placeholderText">
           <string>Enter valid ISBN No.</string>
          </property>
         </widget>
        </item>
        <item row="11" column="0">
         <widget class="QLabel" name="label_20">
          <property name="font">
           <font>
            <family>Times New Roman</family>
            <pointsize>11</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Book Format</string>
          </property>
         </widget>
        </item>
        <item row="11" column="1">
         <widget class="QComboBox" name="book_format">
          <property name="font">
           <font>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">padding:2px;</string>
          </property>
         </widget>
        </item>
        <item row="12" column="0">
         <widget class="QLabel" name="label_15">
          <property name="font">
           <font>
            <family>Times New Roman</family>
            <pointsize>11</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Type of Book </string>
          </property>
         </widget>
        </item>
        <item row="12" column="1">
         <widget class="QComboBox" name="type_of_book">
          <property name="font">
           <font>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">padding:2px;</string>
          </property>
         </widget>
        </item>
        <item row="13" column="0">
         <widget class="QLabel" name="label_16">
          <property name="font">
           <font>
            <family>Times New Roman</family>
            <pointsize>11</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Department</string>
          </property>
         </widget>
        </item>
        <item row="13" column="1">
         <widget class="QComboBox" name="department">
          <property name="font">
           <font>
            <family>Times New Roman</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">padding:2px;</string>
          </property>
         </widget>
        </item>
        <item row="14" column="0">
         <widget class="QLabel" name="label_18">
          <property name="font">
           <font>
            <family>Times New Roman</family>
            <pointsize>11</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Release year</string>
          </property>
         </widget>
        </item>
        <item row="14" column="1">
         <widget class="QComboBox" name="book_releas_year">
          <property name="font">
           <font>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">padding:2px;</string>
          </property>
         </widget>
        </item>
        <item row="15" column="0">
         <widget class="QLabel" name="label_19">
          <property name="font">
           <font>
            <family>Times New Roman</family>
            <pointsize>11</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Edition</string>
          </property>
         </widget>
        </item>
        <item row="15" column="1">
         <widget class="QComboBox" name="book_edition">
          <property name="font">
           <font>
            <pointsize>11</pointsize>
           </font>
          </property>
         </widget>
        </item>
        <item row="7" column="1">
         <widget class="QLineEdit" name="book_author_4">
          <property name="font">
           <font>
            <family>Times New Roman</family>
            <pointsize>11</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">padding:2px;
border-style:solid;
border:2px solid blue;
border-radius:8px;</string>
          </property>
          <property name="placeholderText">
           <string>Fourth Author</string>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="QLineEdit" name="book_author_3">
          <property name="font">
           <font>
            <family>Times New Roman</family>
            <pointsize>11</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">padding:2px;
border-style:solid;
border:2px solid blue;
border-radius:8px;</string>
          </property>
          <property name="placeholderText">
           <string>Third Author</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QLineEdit" name="book_author_2">
          <property name="font">
           <font>
            <family>Times New Roman</family>
            <pointsize>11</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">padding:2px;
border-style:solid;
border:2px solid blue;
border-radius:8px;</string>
          </property>
          <property name="placeholderText">
           <string>Second Author</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QLineEdit" name="book_author_1">
          <property name="font">
           <font>
            <family>Times New Roman</family>
            <pointsize>11</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">padding:2px;
border-style:solid;
border:2px solid blue;
border-radius:8px;</string>
          </property>
          <property name="placeholderText">
           <string>First Author</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="book_keyword">
          <property name="font">
           <font>
            <family>Times New Roman</family>
            <pointsize>11</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">padding:2px;
border-style:solid;
border:2px solid blue;
border-radius:8px;</string>
          </property>
          <property name="placeholderText">
           <string>Enter Keyword</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="bookholics.qrc"/>
 </resources>
 <connections/>
</ui>
